engine/gamesys/src/gamesys/resources/res_glyph_bank.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020-2023 The Defold Foundation
```

Comment:
For most (all?) other res_*.h, we have them in the dmsdk folder, since some extensions might need them.
I think this one is ok to be private (at least for now) since even res_font_map.h doesn't refer to this data, and we have no current use case for making it public.
As long as it builds when using the extender, I'm happy.